beacon/endpoints/rest/schemas/default.py

```diff
@@ -126,6 +148,9 @@ def beacon_individual_v20(row):
         'diseases': row['diseases'],
         'pedigrees': row['pedigrees'],
         'handovers': row['handovers'],
+        #'interventions': row['interventions'],
```

Comment:
In draft.3 of `Individual` schema, these fields exist: `interventions`, `measures` and `exposures` (check [here](https://github.com/ga4gh-beacon/specification-v2-default-schemas/blob/afa5095823c513b54359f929d04c1a2b79cbe2ea/default_individual_schema.yaml#L295)). 
Also, there are 2 missing fields in this implementation: `treatments` and `phenotypicFeatures`.